math/src/main/java/com/powsybl/math/timeseries/AbstractCompressedArrayChunk.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.math.timeseries;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public abstract class AbstractCompressedArrayChunk {
+
+    protected final int offset;
+
+    protected final int uncompressedLength;
+
+    protected final int[] stepLengths;
+
+    public AbstractCompressedArrayChunk(int offset, int uncompressedLength, int[] stepLengths) {
+        this.offset = offset;
+        this.uncompressedLength = uncompressedLength;
```

Comment:
Check for positive values (>=0)

Reply:
See check method just after called in Compressed*ArrayChunk